beacon-chain/p2p/parameter_test.go

```diff
@@ -10,7 +10,6 @@ import (
 
 const (
 	// overlay parameters
-	gossipSubD   = 8  // topic stable mesh target count
```

Comment:
In the original PR, there was a duplicate declaration of entity. I've removed duplicate declaration of `gossipSubD` from test file as `gossip_scoring_params.go` seemed like a good place for that param.

Reply:
Ah good catch, trying to think if we should be having the parameter in a non-test file here. They are declared in the test file as a strict sanity check for gossip parameters. I can instead use the value from the pubsub package directly.